rust-runtime/aws-smithy-client/src/lib.rs

```diff
@@ -78,7 +78,13 @@ pub mod conns {
 
     #[cfg(feature = "native-tls")]
     pub fn native_tls() -> NativeTls {
```

Comment:
I guess returning an error here is too much of a breaking change right?

Reply:
It's also unnecessary - that should never fail since we are building the default configuration with known-good settings.